frontend/src/metabase/visualizations/lib/settings/graph.ts

```diff
@@ -169,10 +184,11 @@ export const GRAPH_DATA_SETTINGS = {
         truncateAfter: 10,
       };
     },
-    getHidden: (series, settings, { transformedSeries }) => {
+    getHidden: (_, settings, extra) => {
       return (
-        settings["graph.dimensions"]?.length < 2 ||
-        transformedSeries.length > MAX_SERIES
+        (settings["graph.dimensions"]
```

Comment:
Well since you pointed this out. This actually changes the current behavior when `graph.dimensions` is undefined.

The question is does that make any difference? Since I'm not totally sure, I'd just coerce it to `0` to match the previous behavior just to be safe. What do you think?

This might relate to the comment I made above as well on `graph.metrics`.